go/src/meguca/parser/commands.go

```diff
@@ -44,11 +46,77 @@ func parseCommand(match []byte, board string) (com common.Command, err error) {
 	// 8ball; select random string from the the 8ball answer array
 	case bytes.Equal(match, []byte("8ball")):
 		com.Type = common.EightBall
-		answers := config.GetBoardConfigs(board).Eightball
+		answers := boardConfig.Eightball
 		if len(answers) != 0 {
 			com.Eightball = answers[randInt(len(answers))]
 		}
 
+	// Increment pyu counter
+	case bytes.Equal(match, []byte("pyu")):
+		if boardConfig.Pyu {
+			com.Type = common.Pyu
+			now := time.Now().UTC()
+			exists, err := db.PyuLimitExists(ip, board)
```

Comment:
Okay, problem. This entire block needs to be done in one transaction to avoid conflicts in case multiple of them run in parallel. See `db.StartTransaction()`.